plugin/src/main/java/org/opensearch/ml/action/prediction/TransportPredictionTaskAction.java

```diff
@@ -135,6 +139,8 @@ public void onResponse(MLModel mlModel) {
                                         } else {
                                             executePredict(mlPredictionTaskRequest, wrappedListener, modelId);
                                         }
+                                    } else if (functionName == FunctionName.REMOTE) {
```

Comment:
Add a feature flag to control enable/disable this auto deployment ?

Reply:
good point, will add one feature flag in the refresh.